tutorials/scripting/gdscript/gdscript_exports.rst

```diff
@@ -244,9 +244,15 @@ Therefore, if you specify an extension of Resource such as:
 The drop-down menu will be limited to AnimationNode and all
 its derived classes.
 
-It must be noted that even if the script is not being run while in the
-editor, the exported properties are still editable. This can be used
-in conjunction with a :ref:`script in "tool" mode <doc_gdscript_tool_mode>`.
+One consideration in using ``@export`` variables for classes that 
+derive from :ref:`Resource <class_Resource>` is that it loads the
+resource when the instance of the script is loaded. This is often
+desirable as you'll likely want to use that resource directly in
+your script. However, if the resource in question is a large
+resource commonly loaded during a game's loading screen (like a
+:ref:`PackedScene <class_PackedScene>` containing a whole level),
+it may be desirable to use ``@export_file`` so you have the file
+path to the resource file and can load it manually when you need it.
```

Comment:
I think it is too easy to understand this as "if your resource is big, use export_file" which I don't really agree with since in most cases the big resource is embedded into a big scene which gets loaded async anyways.

I think this should be a note to make it less conclusive and also an example of a case where it would make sense and were it wouldn't might help to convey that this requires a case by case decision. Examples that come to mind:
- use export_file : a level manager scene which holds the player and instantiates and frees levels as the player transitions through them
- don't use export_file : a big level scene which instantiates some subscenes using a script (for whatever reason :sweat_smile:)